ansible/library/generate_golden_config_db.py

```diff
@@ -199,25 +229,55 @@ def generate_smartswitch_golden_config_db(self):
         ori_config_db = json.loads(out)
         if "DEVICE_METADATA" not in ori_config_db or "localhost" not in ori_config_db["DEVICE_METADATA"]:
             return "{}"
-
         ori_config_db["DEVICE_METADATA"]["localhost"]["subtype"] = "SmartSwitch"
+        hwsku = ori_config_db["DEVICE_METADATA"]["localhost"].get("hwsku", None)
+        platform = ori_config_db["DEVICE_METADATA"]["localhost"].get("platform", None)
+
+        if "FEATURE" not in ori_config_db \
+                or "dhcp_relay" not in ori_config_db["FEATURE"]:
+            return "{}"
+        ori_config_db["FEATURE"]["dhcp_relay"]["state"] = "disabled"
+
+        # Generate INTERFACE table for backplane interfaces
+        if "PORT" not in ori_config_db or "INTERFACE" not in ori_config_db:
+            return "{}"
+
+        if hwsku not in smartswitch_hwsku_config:
+            return "{}"
+
+        if "CHASSIS_MODULE" not in ori_config_db:
+            ori_config_db["CHASSIS_MODULE"] = {}
+            skudir = "/usr/share/sonic/device/{}/{}/".format(platform, hwsku)
+            config_file_path = os.path.join(skudir, "config_db.json")
+            if os.path.exists(config_file_path):
+                with open(config_file_path, "r") as f:
+                    config_data = json.load(f)
+                if "CHASSIS_MODULE" in config_data:
+                    ori_config_db["CHASSIS_MODULE"].update(config_data["CHASSIS_MODULE"])
+        hwsku_config = smartswitch_hwsku_config[hwsku]
+        dpu_num = len(ori_config_db["CHASSIS_MODULE"].keys())
+        smartswitch_hwsku_config[hwsku]['dpu_num'] = int(format(dpu_num))
+        ori_config_db = self.apply_hwsku_config_to_golden_db(
+                                                            hwsku,
+                                                            ori_config_db,
+                                                            hwsku_config,
+                                                            smartswitch_hwsku_config)
+
         gold_config_db = {
-            "DEVICE_METADATA": copy.deepcopy(ori_config_db["DEVICE_METADATA"])
+            "DEVICE_METADATA": copy.deepcopy(ori_config_db["DEVICE_METADATA"]),
+            "FEATURE": copy.deepcopy(ori_config_db["FEATURE"]),
+            "INTERFACE": copy.deepcopy(ori_config_db["INTERFACE"]),
+            "PORT": copy.deepcopy(ori_config_db["PORT"]),
+            "CHASSIS_MODULE": copy.deepcopy(ori_config_db["CHASSIS_MODULE"]),
```

Comment:
Can you share some content of the golden_config_db.json in PR description that it is correctly generated?